ospool-pilot/itb/pilot/where-am-i

```diff
@@ -45,6 +45,18 @@ if [ "$glidein_config" != "NONE" ]; then
     source $add_config_line_source
 fi
 
+function get_glidein_config_value {
+    # extracts a config attribute value from 
+    # $1 is the attribute key
+    CF=$glidein_config
+    if [ "$glidein_config" = "NONE" ]; then
+        CF="$PWD/glidein_config"
+    fi
+    KEY="$1"
+    VALUE=`(cat $CF | grep "^$KEY " | tail -n 1 | sed "s/^$KEY //") 2>/dev/null`
+    echo "$VALUE"
+}
+
```

Comment:
Let's stick this block with the rest of the functions above (before the `if [ "$glidein_config" != "NONE" ]; then` line).